engines/xeen/font.cpp

```diff
@@ -34,15 +35,60 @@ byte FontData::_bgColor;
 bool FontData::_fontReduced;
 Justify FontData::_fontJustify;
 
+Common::Language lang;
+int offset_fntEn;
+int offset_fntEnReduced;
+int offset_fntNonEn;
+int offset_fntNonEnReduced;
+int offset_fntEnW;
+int offset_fntEnReducedW;
+int offset_fntNonEnW;
+int offset_fntNonEnReducedW;
```

Comment:
Can these be moved into font surface? It looks like you're initializing them in the FontSurface constructor anyway, so I don't see them needing to globals.

Reply:
Also remember to use underscores for them as class fields, like "_lang"

Reply:
Oh nevermind, looks like it was already done in a later commit. Good.